go/chat/storage/inbox.go

```diff
@@ -102,14 +126,14 @@ func (i *Inbox) dbKey(uid gregor1.UID) libkb.DbKey {
 	}
 }
 
-func (i *Inbox) readDiskInbox(ctx context.Context, uid gregor1.UID) (inboxDiskData, Error) {
+func (i *Inbox) readDiskInbox(ctx context.Context, uid gregor1.UID, useInMemory bool) (inboxDiskData, Error) {
```

Comment:
I'm not sure I understand, what do you mean "toggled"? Like `false` by default? I wanted the logic inside the function to not have any `!` since it is just easier for me to understand.

Reply:
`useInMemory=false` only in tests. why not have it default to `true` and then call`setUseInMemory(false)` during the tests when needed?

Reply:
I thought it might be useful for something else in the future, the only callers of this are within `Inbox`, and `ReadAll` is also basically only used by tests.